articles/design/using-auth0-with-multi-tenant-apps.md

```diff
@@ -9,50 +9,48 @@ contentType: concept
+* [Use separate Auth0 tenants](#create-separate-auth0-tenants-for-each-customer)
 
-| Tenancy Type | Definition |
-| - | - |
-| Multi-Tenant | One instance, multiple customers |
-| Single-Tenant | One instance, one customer |
+### Use multiple connections
 
-We define **tenant** as a group of users who share access to one particular application instance. One example of this includes a company with multiple employees, all of whom have access to your SaaS offering. 
+With this scenario, you would separate tenants by connection. Each connection would be a different pool of users. 
 
-When you use a multi-tenant setup, one single instance of your SaaS offering would be shared across multiple tenants (or multiple companies), each with its own group of employees. However, each tenant has a dedicated share of that instance, and you can then customize each share to meet the needs of the tenant that's using it. Such customization includes (but isn't limited to) branding, functionality, and access control.
+::: warning
+If you use [Lock](/libraries/lock) in your applications, Lock supports a maximum of **50 Database Connections** per [application](/applications). Enterprise Connections are not affected by this limit. If you use the New Universal Login Experience, Lock is not involved and this limitation therefore does not affect you.
+:::
 
-## Auth0 and multi-tenancy
+While using multiple [Connections](/identityproviders) introduces additional layers of complexity, there are several scenarios where this option might make sense:
+
+* You have different Connection-level requirements, such as varying password policies, for each of your Applications.
+* You have user pools from different Connections. For example, one app may have users providing username/password credentials, while another app handles Enterprise logins.
 
-There are several ways you can handle multi-tenancy with Auth0. When using a single Auth0 tenant for all your customers you can:
+You can call `/authorize` with a specific connection for a user, using the `connection` option in the [Auth0 SPA SDK](/libraries/auth0-spa-js), or by passing a `connection` parameter to the `authorize()` method in [Auth0.js](/libraries/auth0js/v9).
 
-* Use multiple [connections](/connections).
-* Identify different tenants by application (using client IDs).
-* Store tenant details in [app_metadata](/users/concepts/overview-user-metadata).
+### Identify tenants by application
 
-Another approach would be to use separate Auth0 tenants for each of your customers.
+In this scenario, each of your tenants would be a separate application in Auth0. This allows you to configure each slightly differently and to enable or disable connections per application at will, if your customers/tenants have different needs. This will require you to keep track of which tenant each of your users belongs to in your application, and then when logging them in, specify a connection to let them use. 

```

Comment:
You can represent each of your tenants with a separate [application](/applications) in Auth0.

Representing each of your tenants with an application allows you to configure each one differently. You can also enable/disable [connections](/identityproviders) for individual applications if your tenants have varying requirements. Doing so, however, requires you to track the tenants to which your users belong in your application. Then, when they log in, you'll need to specify the connection they use.